src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -2420,129 +2523,167 @@ subroutine atm_advance_acoustic_step_work(nCells, nEdges, nCellsSolve, cellStart
            if (cell1 <= nCellsSolve .or. cell2 <= nCellsSolve ) then
 
 !DIR$ IVDEP
+              !$acc loop vector
               do k=1,nVertLevels
                  ru_p(k,iEdge) = dts*tend_ru(k,iEdge)
-              end do
-!DIR$ IVDEP
-              do k=1,nVertLevels
-                 ruAvg(k,iEdge) = ru_p(k,iEdge)                 
+                 ruAvg(k,iEdge) = ru_p(k,iEdge)
               end do
 
            end if ! end test for block-owned cells
 
         end do ! end loop over edges
+        !$acc end parallel
 
       end if ! test for first acoustic step
 
       if (small_step == 1) then  ! initialize here on first small timestep.
+         !$acc parallel default(present)
+<<<<<<< HEAD
+         !$acc loop gang worker vector collapse(2)
+=======
+         !$acc loop collapse(2)
+>>>>>>> d7109c12a (fixup! Add acc data movement to atm_advance_acoustic_step_work)
```

Comment:
There's a merge conflict here fyi.

Reply:
Thanks for spotting it. I'll get that sorted.

Reply:
Addressed by force-push [68253c3](https://github.com/MPAS-Dev/MPAS-Model/commit/68253c37370aecd6e58966569ad217c03d041042) to [6e40864](https://github.com/MPAS-Dev/MPAS-Model/commit/6e40864b4974f57ce6fbbfe30437cf16c291260c).